python/lsst/daf/butler/tests/hybrid_butler.py

```diff
@@ -72,6 +72,9 @@ def __new__(cls, remote_butler: RemoteButler, direct_butler: DirectButler) -> Hy
         self = cast(HybridButler, super().__new__(cls))
         self._remote_butler = remote_butler
         self._direct_butler = direct_butler
+        # Force the dimension universes to match remote and direct.
+        with self._remote_butler._cache.access() as cache:
+            cache.dimensions = self._direct_butler.dimensions
```

Comment:
I guess this should work OK for tests, but does this expect that local universe and remote universe are identical? What happens if we add a test when they are different? Would it be better to test here that they are "equal" (namespace and version number)?

Reply:
hybrid butler is specifically designed for testing but remote butler is specifically designed not to use the main universe cache so this didn't seem like a problem. I could add a `DimesnsionUniverse.__eq__` method instead -- maybe we didn't have one originally because we didn't have formal versioning but now we have decided that if someone edits the universe yaml without changing the version number that we are going to have bigger problems anyhow so just assume it's enough.